k8s/secrets.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-kind: ConfigMap
+kind: Secret
 metadata:
-  name: django-config
-data:
-    # n.b., these are debug values / non-secret secrets
+  name: secrets-env

```

Comment:
I think this should be `dt-secrets-env`? (If I understand correctly, it's just for illustration)

Reply:
No, as Kustomize will prefix it with `dt` upon creation.